CesiumGltf/include/CesiumGltf/PropertyView.h

```diff
@@ -890,15 +929,6 @@ template <> class PropertyView<bool> {
       const PropertyTableProperty& /*property*/)
       : PropertyView(classProperty) {}
 
-  /**
-   * @brief Constructs a property instance from a property texture property and
-   * its class definition.
-   */
-  PropertyView(
```

Comment:
Removed this constructor where it didn't make sense (not all property types can be represented by property textures).